service/web/tabs/app/src/containers/ServiceContainer.tsx

```diff
@@ -17,12 +17,13 @@ class ServiceContainer extends React.Component<
   IState
 > {
   private service: string = (this.props as any).match.params.service
-  private backfillRunsTag: string = `${this.service}::BackfillRuns`
+  private variant: string = (this.props as any).match.params.variant
+  private backfillRunsTag: string = `${this.service}::${this.variant}::BackfillRuns`
 
   componentDidMount() {
     this.props.simpleNetworkGet(
       this.backfillRunsTag,
-      `/services/${this.service}/backfill-runs`
+      `/services/${this.service}/backfill-runs?variant=${this.variant}`
```

Comment:
The mixing in the ways of specifying the url seems odd. We should be consistent.